src/app/pages/apptrust/apptrust.page.html

```diff
@@ -9,6 +9,40 @@
   </ion-toolbar>
 </ion-header>
 
-<ion-content >
-  
-</ion-content>
+<ion-content>
+  <ion-card>
+    <ion-card-header>
+      <ion-card-title>Access the Mobile Security Service</ion-card-title>
+      <ion-card-subtitle>Use the Button to access the Mobile Security Service </ion-card-subtitle>
```

Comment:
Buttons were usually for actions. I will suggest the link or simply card item.  Asking for opinion our Lead Designer:  @piccolazucca